android/gradle.properties

```diff
@@ -6,7 +6,7 @@ org.gradle.caching=true
 org.gradle.jvmargs=-Xmx1024m -Xms256m
 android.useAndroidX=true
 android.native.buildOutput=verbose
-android.nonTransitiveRClass=false
+android.nonTransitiveRClass=true
```

Comment:
> The nonTransitiveRClass flag in the gradle.properties file controls the R class behavior. From AGP 8.0, it’s true when not specified, and thus becomes the default.

We don't have to even specify it, right? It's already `true` by default.

Reply:
Let's leave it for clarity.